sympy/physics/mechanics/kane.py

```diff
@@ -63,9 +78,27 @@ class KanesMethod(_Methods):
     are given in the explicit form by default. In other words, the kinematic
     mass matrix is $\mathbf{k_{k\dot{q}}} = \mathbf{I}$.
     In order to get the implicit form of those matrices/vectors, you can set the
-    ``explicit_kinematics`` attribute to ``False``. So $\mathbf{k_{k\dot{q}}}$ is not
-    necessarily an identity matrix. This can provide more compact equations for
-    non-simple kinematics (see #22626).
+    ``explicit_kinematics`` attribute to ``False``. So $\mathbf{k_{k\dot{q}}}$
+    is not necessarily an identity matrix. This can provide more compact
+    equations for non-simple kinematics.
+
+    Two linear solvers can be supplied to ``KanesMethod``: one for solving the
+    kinematic differential equations and one to solve the velocity constraints.
+    Both of these sets of equations can be expressed as a linear system ``Ax = rhs``,
+    which have to be solved in order to obtain the equations of motion.
+
+    The default solver, LU solve, results in the fastest execution time after
```

Comment:
I don't think we should mention lambdify here. It does results in faster equation of motion formulation symbolically.

Reply:
Can mention that it results in a relatively low number of operations.